data/difficulties.dat

```diff
@@ -0,0 +1,41 @@
+[very-easy]
```

Comment:
This file is unrelated to i18n change, or isn't it? Maybe you can remove it from this PR and re-add it in a new PR?